helpers.py

```diff
@@ -305,6 +305,16 @@ def blend_images(images, margin, zoom, resampling_func):
 
     return images
 
+def resize_all_image(images, resize_factor, resampling_func):
+    if isclose(resize_factor, 1.0):
```

Comment:
Is there any particular reason for using isclose? It seems to me that == 1.0 should be enough.

Reply:
It's just a habit of never comparing floats directly. Here it's indeed fine, I'll revert it...